blazor/diagram/getting-started/server-side-application.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Getting Started with Diagram Component in Blazor Server App | Syncfusion
+description: Learn how to getting started with Diagram control in Blazor Server-side application
```

Comment:
description: Learn how to get started with Diagram control in the Blazor Server-side application.